capa/main.py

```diff
@@ -1038,6 +1035,11 @@ def handle_common_args(args):
             logger.debug("-" * 80)
 
             sigs_path = os.path.join(get_default_root(), "sigs")
+            if not os.path.exists(sigs_path):
+                raise IOError(
+                    "Please install the signatures first: "
+                    "https://github.com/mandiant/capa/blob/master/doc/installation.md#method-2-using-capa-as-a-python-library."
+                )
```

Comment:
```suggestion
            if not os.path.exists(sigs_path):
                logger.error(
                    "Using default signature path, but it doesn't exist. Please install the signatures first: "
                    "https://github.com/mandiant/capa/blob/master/doc/installation.md#method-2-using-capa-as-a-python-library."
                )
                raise IOError(f"signatures path {sigs_path} does not exist or cannot be accessed")
```

use the logger to show messages to the user, while the exception should contain the problem itself.